test/models/parser/youtube_item_test.rb

```diff
@@ -15,8 +15,8 @@ class YoutubeItemIntegrationTest < ActiveSupport::TestCase
   end
 
   test "should store oembed data of a youtube item" do
-    RequestHelper.stubs(:get_html).returns(Nokogiri::HTML("<link rel='alternate' type='application/json+oembed' href='http://www.youtube.com/oembed?format=json&amp;url=https%3A%2F%2Fwww.youtube.com%2Fwatch%3Fv%3DmtLxD7r4BZQ' title='RubyConf Portugal 2016 - Best Of'>"))
-    m = create_media url: 'https://www.youtube.com/watch?v=mtLxD7r4BZQ'
+    RequestHelper.stubs(:get_html).returns(Nokogiri::HTML("<link rel='alternate' type='application/json+oembed' href='http://www.youtube.com/oembed?format=json&amp;url=https%3A%2F%2Fwww.youtube.com%2Fwatch%3Fv%3DfHatsiQvDWc' title='RubyConf Portugal 2016 - Best Of'>"))
```

Comment:
@hartsick one thing to add here: this link `http://www.youtube.com/oembed?format=json&amp;url=https%3A%2F%2Fwww.youtube.com%2Fwatch%3Fv%3DfHatsiQvDWc` goes nowhere, but if we change it to `https://www.youtube.com/oembed?format=json&url=https%3A%2F%2Fwww.youtube.com%2Fwatch%3Fv%3DfHatsiQvDWc` it works properly.

The change is `&amp;` to `&`

For the test the one that goes nowhere worked though 🤔

Reply:
I'm not sure what to make of that last point, but making sure the URL uses the proper characters (& instead of &amp;) makes sense to me!